src/renderer/src/hooks/useConfigFileImporter.ts

```diff
@@ -0,0 +1,30 @@
+import { useState } from 'react'
+
+export function useConfigFileImporter() {
```

Comment:
with some changes to how the file opening mechanics work (as detailed by my other comment), I can see this being simplified to a light wrapper for the "runtime" in the form of a react-query mutation. something along the lines of:

```ts
export function useSelectConfigFile() {
  return useMutation({
    mutationFn: async () => {
      // selectFile yet to be implemented
      return window.runtime.selectFile({
        extensionFilters: ['comapeocat']
      })
    }
  })
}
```